zkevm-circuits/src/state_circuit.rs

```diff
@@ -474,6 +487,7 @@ impl<F: Field> SubCircuit<F> for StateCircuit<F> {
                             offset,
                             || Value::known(f),
                         )?;
+                        region.name_column(|| "STATE_test_override", advice_column);
```

Comment:
You should move this outside of the for loop to avoid unnecessary repeated calls.

Reply:
There is a for loop to iterate all columns. So, I guess it's fine to keep here. However, I'm not sure about that  we need to add annotations in the testing circuit?

Reply:
Ah I didn't look deep enough into this.  There should be no annotations added here.  It's not because this is a testing circuit, but because the `overrides` field contains a list of existing columns + offsets for which the assigned value needs to be overridden (to do negative testing).
So having annotations here would override the original annotations :sweat_smile:

Reply:
fixed in [d4a9d84](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/d4a9d8498129a44ce31833fc61e4857e231f4d5c)